spec/rubocop/cop/style/mutable_constant_spec.rb

```diff
@@ -157,6 +194,34 @@
           RUBY
         end
       end
+
+      context 'when using shareable_constant_value' do
+        it_behaves_like 'literals that are frozen', '# shareable_constant_value: literal'
+        it_behaves_like 'literals that are frozen', '# shareable_constant_value: experimental_everything'
+        it_behaves_like 'literals that are frozen', '# shareable_constant_value: experimental_copy'
+        it_behaves_like 'literals that are not frozen', '# shareable_constant_value: none'
+      end
+
+      it 'raises offense when shareable_constant_value is specified as an inline comment' do
+        expect_offense(<<~RUBY)
+          X = [1, 2, 3] # shareable_constant_value: literal
```

Comment:
At some point we'll need a lint cop to check for this. :D

Reply:
Makes, sense. Since it's an experimental feature, maybe we need not do it right away.